css/style.css

```diff
@@ -15,5 +14,195 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+h1 {
+  margin: auto;
+  margin-top: auto;
+  font-size: 40px;
+}
+h2 {
+  margin: auto;
+  margin-top: auto;
+  font-size: 20px;
+}
+h3 {
+  text-align: center;
+  font-size: 24px;
+  top: 30%;
+}
+
+#navbar-button {
+  margin-left: 25px;
+  margin-right: 25px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  align-content: space-between;
+  margin: 15px;
+}
+
+.navbar-logo {
+  display: flex;
+  margin: auto;
+  width: 2%;
+  margin-left: 200px;
+}
+
+.navbar-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  list-style-type: none;
+  font-size: smaller;
+  margin-right: 60px;
+}
 
+a:hover {
```

Comment:
Great 👍 
This will change colour for mouse users who hover. 
Which pseudo-class could you add to also change the colour for a keyboard user who has tabbed onto the link?